napari/benchmarks/benchmark_shapes_layer.py

```diff
@@ -27,48 +27,140 @@
     from napari.benchmarks.utils import Skip
 
 
-class Shapes2DSuite:
+class BackendType(Enum):
```

Comment:
@Czaki Based on [this comment](https://github.com/napari/napari/pull/7747#issuecomment-2766109565) in #7747, I think it might be good to still include partsegcore here, because any users who turned it on and installed partsegcore in 0.5.6 will suddenly lose acceleration when they upgrade (because they won't know to install bermuda). And anyway it would be good to be able to directly compare bermuda and partsegcore, no?

Reply:
The #7747 itself, when You run benchmark on it will be comparison. But I will restore option to use PartSegCore.